util/fibers/uring_proactor.cc

```diff
@@ -717,24 +716,23 @@ void UringProactor::MainLoop(detail::Scheduler* scheduler) {
       // do now execute any syscalls. Therefore we count how many completions we handled
       // and reap the same amount.
       ReapCompletions(cqe_count, cqes, dispatcher);
+
+      if (HasSleepFibersStarved()) {
+        ProcessSleepFibers(scheduler);
```

Comment:
Since you're checking `HasSleepFibersStarved()` inside `ProcessSleepFibers()`, I'd remove the above check (or the one inside).

Reply:
i do not check for HasSleepFibersStarved there

Reply:
Sorry, my bad